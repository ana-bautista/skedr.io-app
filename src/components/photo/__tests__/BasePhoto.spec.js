import { shallowMount } from '@vue/test-utils'
import Vue from 'vue'
import Vuetify from 'vuetify'
import Comp from '../BasePhoto.vue'

Vue.use(Vuetify)

describe('BasePhoto component', () => {
  it('prints base component', () => {
    const wrapper = shallowMount(Comp, {
      propsData: { url: 'url1', lazyUrl: 'lazyUrl', isPrivate: true },
      slots: { header: '<span>header slot</span>', footer: '<span>footer slot</span>' }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
