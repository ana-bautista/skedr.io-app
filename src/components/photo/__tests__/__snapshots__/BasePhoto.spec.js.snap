// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasePhoto component prints base component 1`] = `
<v-sheet-stub tag="div"><span>header slot</span>
  <v-img-stub height="205" aspectratio="1" src="url1" lazysrc="url1" position="center center" transition="fade-transition" class="grey lighten-2 img">
    <v-layout-stub tag="div" fill-height="" align-center="" justify-center="" ma-0="">
      <v-progress-circular-stub color="grey lighten-5" indeterminate="true" rotate="0" size="32" width="4" value="0"></v-progress-circular-stub>
    </v-layout-stub>
  </v-img-stub>
  <v-tooltip-stub opendelay="200" closedelay="200" lazy="true" contentclass="" fixed="true" left="true" maxwidth="auto" nudgebottom="0" nudgeleft="0" nudgeright="0" nudgetop="0" nudgewidth="0" debounce="0" tag="span" class="non-public">
    <v-icon-stub large="" class="warning--text">lock</v-icon-stub> <span>Private image</span>
  </v-tooltip-stub> <span>footer slot</span></v-sheet-stub>
`;
